sympy/physics/control/lti.py

```diff
@@ -1079,78 +1328,51 @@ def doit(self, **kwargs):
-            arg_list.append(other)
+        if not isinstance(other, LinearTimeInvariant):
+             return NotImplemented
 
-            return Parallel(*arg_list)
-        elif isinstance(other, Parallel):
-            if not self.var == other.var:
-                raise ValueError("All the transfer functions should use the same complex variable "
-                    "of the Laplace transform.")
+        if not other.is_SISO:
+            raise TypeError("Cannot add SISO and MIMO systems together.")
+
+        if isinstance(other, Parallel):
             self_arg_list = list(self.args)
             other_arg_list = list(other.args)
-            for elem in other_arg_list:
-                self_arg_list.append(elem)
+            return Parallel(*self_arg_list, *other_arg_list)
 
-            return Parallel(*self_arg_list)
-        else:
-            raise ValueError("This transfer function expression is invalid.")
+        self_arg_list = list(self.args)
+        return Parallel(*self_arg_list, other)
 
-    def __sub__(self, other):
-        if isinstance(other, (TransferFunction, Series)):
-            if not self.var == other.var:
-                raise ValueError("All the transfer functions should use the same complex variable "
-                    "of the Laplace transform.")
-            arg_list = list(self.args)
-            arg_list.append(-other)
+    __radd__ = __add__
 
-            return Parallel(*arg_list)
-        elif isinstance(other, Parallel):
-            if not self.var == other.var:
-                raise ValueError("All the transfer functions should use the same complex variable "
-                    "of the Laplace transform.")
-            self_arg_list = list(self.args)
-            other_arg_list = list(other.args)
-            for elem in other_arg_list:
-                self_arg_list.append(-elem)
+    def __sub__(self, other):
+        if not isinstance(other, SISOLinearTimeInvariant):
+             return NotImplemented
+        return self + (-other)
 
-            return Parallel(*self_arg_list)
-        else:
-            raise ValueError("This transfer function expression is invalid.")
+    def __rsub__(self, other):
+        return -self + other
 
     def __mul__(self, other):
-        if isinstance(other, (TransferFunction, Parallel)):
-            if not self.var == other.var:
-                raise ValueError("All the transfer functions should use the same complex variable "
-                    "of the Laplace transform.")
-            return Series(self, other)
-        elif isinstance(other, Series):
-            if not self.var == other.var:
-                raise ValueError("All the transfer functions should use the same complex variable "
-                    "of the Laplace transform.")
+        if not isinstance(other, LinearTimeInvariant):
+             return NotImplemented
+
+        if not other.is_SISO:
+            raise TypeError("Cannot multiply SISO and MIMO systems together.")
```

Comment:
I guess this is analogous to multiplying a scalar and a matrix. Should that be allowed?

Also is it important to show this particular error message because this second check wouldn't be needed if the first check was for SISOLinearTimeInvariant

Reply:
> I guess this is analogous to multiplying a scalar and a matrix. Should that be allowed?
> 
We can make that work later on if needed.

Reply:
> I guess this is analogous to multiplying a scalar and a matrix. Should that be allowed?

We can allow that, but I don't think that it would have any physical significance.

Reply:
Taken from [`SystemsModelParallelConnect`](https://reference.wolfram.com/language/ref/SystemsModelParallelConnect.html):

Add a disturbance model to the output of a system:
![image](https://user-images.githubusercontent.com/43007653/127840848-0c7a7ee8-9513-4a30-a35c-75716221feb2.png)
![image](https://user-images.githubusercontent.com/43007653/127840870-3fd2ece5-a701-4851-9ec5-05e4bb12b970.png)
![image](https://user-images.githubusercontent.com/43007653/127840908-567ac3a1-c321-4eee-ab69-05af4c91c157.png)